FB.py

```diff
@@ -0,0 +1,82 @@
+import unittest
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import Select
+from selenium.common.exceptions import TimeoutException
+import time
+class SignUpTest(unittest.TestCase):
+    def setUp(self):
+        self.driver= webdriver.Firefox(executable_path="/home/ruttaba/PycharmProjects/geckodriver")
+        self.driver.get("https://www.facebook.com/")
+        self.assertIn("Facebook", self.driver.title)
```

Comment:
@ruttaba "setUp" method is not for assertions, it is just for setup. Write separate test if you want to assert this.